tests/integration/azure/test_credentials.py

```diff
@@ -56,21 +58,28 @@ def extract_test_info(ws, env_or_skip, make_random):
 
 
 @pytest.fixture
-def run_migration(ws, sql_backend):
+def run_migration(sql_backend, inventory_schema):
     def inner(
+        ws: WorkspaceClient,
         test_info: MigrationTestInfo,
         credentials: set[str] | None = None,
         read_only=False,
         migrate_service_principals: str = "Yes",
         create_access_connectors: str = "No",
+        azurerm: AzureResources | None = None,
+        resource_permissions: AzureResourcePermissions | None = None,
```

Comment:
not blocking: the more this method grows, the more i think we need `TestCredentialMigrationContext`, like the others we have